mcweb/backend/sources/tasks.py

```diff
@@ -474,13 +481,12 @@ def _get_min_update_date():
 @background(queue=SYSTEM_SLOW)
 def update_source_language(provider_name:str, batch_size: int = 100 ) -> None:
     six_months_ago = _get_min_update_date()
```

Comment:
If we're not filtering `modified_at__lt=six_months_ago`, is six_months_ago still used?